pkg/acsengine/defaults.go

```diff
@@ -584,8 +591,22 @@ func setMasterProfileDefaults(a *api.Properties, isUpgrade bool) {
 	}
 }
 
-// setVMSSDefaults
-func setVMSSDefaults(a *api.Properties) {
+// setVMSSDefaultsForMasters
+func setVMSSDefaultsForMasters(a *api.Properties) {
+	if a.MasterProfile.Count > 100 {
```

Comment:
The only supported values for `MasterProfile.Count` are 1, 3 and 5 so this is unreachable code